src/Microsoft.Health.Fhir.SqlServer/Features/Search/Expressions/Visitors/QueryGenerators/SqlQueryGenerator.cs

```diff
@@ -21,7 +22,13 @@ namespace Microsoft.Health.Fhir.SqlServer.Features.Search.Expressions.Visitors.Q
     internal class SqlQueryGenerator : DefaultExpressionVisitor<SearchOptions, object>, ISqlExpressionVisitor<SearchOptions, object>
     {
         private string _cteMainSelect; // This is represents the CTE that is the main selector for use with includes
-        private List<string> _includeCtes;
+        private List<string> _includeCteIds;
```

Comment:
can this be a local variable?

Reply:
I don't see how.  It is used by all expression visits. .. Unless I don;t understand your question

Reply:
I don;t see how. It accumulates the ctes in each visit... unless I didn;t understand what you mean